packages/frontend/app/components/program-year/objectives.js

```diff
@@ -21,9 +20,8 @@ export default class ProgramYearObjectivesComponent extends Component {
     return this.objectiveCount > 0;
   }
 
-  @action
-  load(element, [programYear]) {
-    this.objectiveCount = programYear.hasMany('programYearObjectives').ids().length;
+  get objectivesCount() {
```

Comment:
typo here, should be `objectiveCount`

```suggestion
  get objectiveCount() {
```